tests/interfaces/messages/messages/messages-get.spec.ts

```diff
@@ -0,0 +1,98 @@
+import type { MessagesGetMessage } from '../../../../src/index.js';
+
+import { expect } from 'chai';
+import { Jws } from '../../../../src/index.js';
+import { Message } from '../../../../src/core/message.js';
+import { MessagesGet } from '../../../../src/index.js';
+import { TestDataGenerator } from '../../../utils/test-data-generator.js';
+
+describe('MessagesGet Message', () => {
+  describe('create', () => {
+    it('creates a MessagesGet message', async () => {
+      const { requester, message } = await TestDataGenerator.generateRecordsWrite();
+      const messageCid = await Message.getCid(message);
+
+      const messagesGet = await MessagesGet.create({
+        authorizationSignatureInput : await Jws.createSignatureInput(requester),
+        messageCids                 : [messageCid]
+      });
+
+      expect(messagesGet.message.authorization).to.exist;
+      expect(messagesGet.message.descriptor).to.exist;
+      expect(messagesGet.message.descriptor.messageCids.length).to.equal(1);
+      expect(messagesGet.message.descriptor.messageCids).to.include(messageCid);
+    });
+
+
+    it('throws an error if at least 1 message cid isnt provided', async () => {
+      const alice = await TestDataGenerator.generatePersona();
+
+      try {
+        await MessagesGet.create({
+          authorizationSignatureInput : await Jws.createSignatureInput(alice),
+          messageCids                 : []
+        });
+
+        expect.fail();
+      } catch (e: any) {
+        expect(e.message).to.include('messageCids');
```

Comment:
nit: include more of the expected error message here. Makes it easier to CMD+f what I broke if I ever break this test one day.

Reply:
good call. this one specifically is auto-generated by AJV, our json schema parser/validator. still went ahead and included fuller error messages for all tests. In this case, hopefully the fuller error message will maaaybe help indicate that it's coming from json schema. went ahead and called that out in a comment